asserts/signtool/sign.go

```diff
@@ -47,6 +55,20 @@ func Sign(opts *Options, keypairMgr asserts.KeypairManager) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot parse the assertion input as JSON: %v", err)
 	}
+
+	for name, value := range opts.Complement {
+		if v, ok := headers[name]; ok {
+			if name == "type" {
+				if v != value {
+					return nil, fmt.Errorf("repeated assertion type does not match")
```

Comment:
(nitpick) maybe `complemented assertion type does not match` here?

Reply:
for the user of a tool using this "complemented" likely doesn't mean much, it's an internal concept, repeated type  specifically hopefully is more understandable